.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
Why do we want all those node versions ?

Reply:
Right now clients of this package use v14 and later on it'll use v20. I've added the in between LTS versions for completeness.